rpcserver.go

```diff
@@ -2356,9 +2398,7 @@ func unmarshalLeafKey(key *unirpc.AssetKey) (universe.BaseKey, error) {
 			return baseKey, err
 		}
 
-		pubKey, err := schnorr.ParsePubKey(
-			scriptKeyBytes,
-		)
+		pubKey, err := parseUserScriptKey(scriptKeyBytes)
```

Comment:
Hmm, in some cases we actually still want the 32-byte key here, though I think we may just lean towards that serializtaion late rdown the call path. Will double check those areas.

Reply:
I think this doesn't break currently because we default to using script keys tweaked BIP 86 style? But could break with a user-supplied script key.